p2p/shard_syncer.py

```diff
@@ -192,6 +188,8 @@ def msg_queue_maxsize(self) -> int:
         # to header existence at the moment. In the future we won't transfer collations anyway but
         # only collation bodies (headers are retrieved from the main chain), so there's no need to
         # add this at the moment.
+        from trinity.protocol.les.utils import gen_request_id
```

Comment:
Cross-package utils import?

Reply:
temporary for the duration of the migration.